src/renderer/components/_molecules/project-tree/index.tsx

```
@@ -114,9 +114,8 @@ const ProjectTreeBranch = ({ branchTarget, children, ...res }: IProjectTreeBranc
             'truncate font-caption text-xs font-normal text-neutral-850 dark:text-neutral-300',
             branchIsOpen && 'font-medium text-neutral-1000 dark:text-white',
           )}
-        >
-          {label}
-        </span>
+          dangerouslySetInnerHTML={{ __html: label || '' }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)